README.md

```diff
@@ -19,9 +19,9 @@ npm install --save-dev @askui/askui-reporters
 
 Detailed examples on how to use the reporters are provided in this README.
 
-### Allure Reporter
+### AskUIAllureStepReporter
 
-> ❗️ Do **NOT** configure multiple reporters as this will break your project in unexpected ways! 
+> ❗️ Do **NOT** configure multiple step reporters in the `UiControlClient` in `jest.setup.ts` as this will break your project in unexpected ways!
```

Comment:
FYI:  Won't be true anymore soon: https://github.com/askui/askui/pull/562

Reply:
Thx for the explanation! Really appreciated ❤️